crates/meroctl/src/cli/context.rs

```diff
@@ -64,6 +65,8 @@ pub enum ContextSubCommands {
     Update(UpdateCommand),
     Identity(ContextIdentityCommand),
     Alias(ContextAliasCommand),
+    #[command(name = "use")]
+    Use(UseCommand),
```

Comment:
this attribute line is not necessary since the command name is already the variant name

```suggestion
    Use(UseCommand),
```